webbpsf/optical_budget.py

```diff
@@ -344,7 +346,7 @@ def visualize_wfe_budget(inst, slew_delta_time=14 * u.day, slew_case='EOL', ptt_
     ax.text(
         0.96,
         0.92,
-        f'45$^\circ$, {slew_delta_time}',
+        f'45$^\circ$, {slew_delta_time}', # noqa
```

Comment:
actually I think this should be fixed rather than noqa'ed. Using a `\\` rather than a single `\` works to escape the back slash properly. (Both single and double backslashes yield the desired output in plot text, which I just verified.)

Reply:
It hought that `\circ` created a special symbol.  You are saying that `\\circ` yields the same result? If so, happy to change all these #noqa as there are plenty of them

Reply:
This is subtle about which piece of code is parsing the string. 

`\circ` în _LaTeX_ (invoked via Matplotlib) does yield the special symbol. But in _python string parsing_ by the python interpreter, for f strings any backslash gets interpreted as indicating an escape sequence, so it checks for `\c` which isn't a valid escape sequence, then falls back to . Passing the python interpreter the double backslash results in a single backslash being passed to LaTeX via Matplotlib.

Reply:
Awesome, I haven't used latex and was researching it, I appreciate the insight.